mmdet3d/models/detectors/imvotenet.py

```diff
@@ -1,51 +1,129 @@
+import numpy as np
+import torch
 from torch import nn as nn
+from torch.nn import functional as F
 
-from mmdet.models import DETECTORS, build_backbone, build_head, build_neck
+from mmdet3d.core import bbox3d2result, merge_aug_bboxes_3d
+from mmdet.models import DETECTORS
+from .. import builder
 from .base import Base3DDetector
 
 
+class ImageMLPModule(nn.Module):
+
+    def __init__(self, input_dim=18, image_hidden_dim=256):
+        super().__init__()
+        self.img_feat_conv1 = nn.Conv1d(input_dim, image_hidden_dim, 1)
```

Comment:
May use conv_cfg and norm_cfg with build_conv_layer and build_norm_layer in case users want to try GN or other layers.